test.nix

```diff
@@ -0,0 +1,54 @@
+{ nixpkgs ? import ./nixpkgs.nix
+, pkgs ? import nixpkgs {}
+}:
+let
+  python = pkgs.python3.withPackages(ps: with ps; [kafka-python]);
+  make-test = import "${nixpkgs}/nixos/tests/make-test-python.nix";
+  test = make-test {
+    name = "kafka-slurp-test";
+    nodes = {
+      slurp = {
+        networking.enableIPv6 = false;
```

Comment:
Add in-code-comment to explain why this is necessary, it's surprising otherwise.